other/cmr-exchange/service-bridge/resources/config/cmr-opendap/config.edn

```diff
@@ -1,102 +1,85 @@
-     :web {
-       :route-keys [:httpd :route-fns]
-       :api-route-key :api
-       :site-route-key :site}}}
+                          :public {:protocol "https"
+                                   :host "cmr.sit.earthdata.nasa.gov"}
+                          :relative {:root {:url "/service-bridge"}}}}
+       :search {:protocol "https"
+                :host "cmr.sit.earthdata.nasa.gov"
+                :relative {:root {:url "/search"}}}}
+ :giovanni {:search {:protocol "https"
+                     :host "giovanni.gsfc.nasa.gov"
+                     :relative {:root {:url "/giovanni"}}}}
+ :edsc {:search {:protocol "https"
+                 :host "search.sit.earthdata.nasa.gov"
+                 :relative {:root {:url "/search"}}}}
+ :httpd {:port 3013
+         :entry-point-fn cmr.opendap.app.core/main
+         :route-fns {:api cmr.opendap.app.routes.rest.core/all
+                     :site cmr.opendap.app.routes.site/all}
+         :pages {:default-title "CMR Service-Bridge"}
+         :base-url "/service-bridge"
+         :base-url-fn cmr.opendap.components.config/opendap-base-url
+         :docs "resources/public/docs"
+         :assets "resources/public/assets"
+         :rest-docs
+         {:base-url-template "%%BASE_URL%%"
+          :source "resources/docs/rest-api/index.md"
+          :outdir "resources/public/docs/service-bridge/docs/current/rest-api"}
+         :skip-static #{"/service-bridge/docs"}
+         :replace-base-url #{"/service-bridge/docs/current/rest-api"
+                             "/service-bridge/docs/current/reference/2000-usage.html"}
+         :index-dirs #{"/service-bridge/docs/1.0.0/marginalia"
+                       "/service-bridge/docs/1.0.0/reference"
+                       "/service-bridge/docs/1.0.0/rest-api"
+                       "/service-bridge/docs/1.5.0/marginalia"
+                       "/service-bridge/docs/1.5.0/reference"
+                       "/service-bridge/docs/1.5.0/rest-api"
+                       "/service-bridge/docs/current/marginalia"
+                       "/service-bridge/docs/current/reference"
+                       "/service-bridge/docs/current/rest-api"}
+         :streaming {:timeout 20000 ; milliseconds (20 seconds * 1000)
+                     :heartbeat 200}} ; milliseconds
+ :logging {:level :trace
+           :nss [cmr org.httpkit]
+           :color false}
+ :mission-control {:messaging-type :core-async}
+ :plugin {:jarfiles {;; The reducer factory function must take two args: plugin name and plugin
+                     ;; type -- both of type ^String. The default reducer checks for an exact
+                     ;; string match.
+                     :reducer-factory cmr.plugin.jar.core/create-has-plugin-type-reducer}
+          :registry {:default {:plugin-name "CMR-Plugin"
+                               :plugin-type "service-bridge-app"
+                               ;; The fowllowing needs to be an in-JAR path to a plugin's config file,
```

Comment:
Minor typo
```suggestion
                               ;; The following needs to be an in-JAR path to a plugin's config file,
```

Reply:
I'll pick it up in my other PR to save time